server/middlewares/UserValidation.js

```diff
@@ -199,6 +199,62 @@ class UserValidation {
     req.body.fullName = `${firstName} ${lastName.join('')}`;
   }
 
+  /**
+    * @description - This method validates social type
+    * @param {object} req - The request object
+    * @returns {null} - returns nothing
+    * @memberOf UserValidation
+    * @static
+    */
+  static validateSocialType(req) {
+    req.checkBody(
+      'socialType',
+      'Please provide a social type e.g facebook'
+    ).exists();
+    req.checkBody(
+      'socialType',
+      'social type must be facebook, twitter or google'
+    ).custom((socialType) => {
+      if (
+        socialType === 'twitter'
+        || socialType === 'facebook'
+        || socialType === 'google'
+      ) return true;
+    });
+  }
+
+  /**
+    * @description - This method validates social id
+    * @param {object} req - The request object
+    * @returns {null} - returns nothing
+    * @memberOf UserValidation
+    * @static
+    */
+  static validateSocialId(req) {
+    req.checkBody(
+      'socialId',
+      'Please provide a socialId e.g 730184790124'
+    ).exists();
```

Comment:
make this `.exists().notEmpty()` to validate against empty strings

Reply:
mkaes sense